src/store/store.js

```diff
@@ -0,0 +1,18 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import loginReducer from './login/reducer';
+import registerReducer from '.register/reducer';
+import thunkMiddleware from './redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
```

Comment:
This is super nit picky but a good practice is to have your imports be (dotted lines indicate a blank line) :

from libraries such as redux and react
...
other modules in your application
... 
local imports such as your './login/reducer